src/components/ButtonSidebar.vue

```diff
@@ -1,15 +1,20 @@
 <script setup lang="ts">
-defineProps<{
-  isDisabled?: boolean;
-}>();
+withDefaults(
```

Comment:
This component is somewhat unclear:

1.  of labeling it as a "sidebar" button, it would be more appropriate to call it a "fully rounded button," given its usage in various locations. A name change should be considered.
2. If the component is referred to as a button, it should use the proper <button> element instead of <a> or both. Alternatively, if it functions as a link, it might be better to rename it as "link."
3. To reduce redundancy, it might be beneficial to refactor this component into either the BaseButton or BaseLink component, depending on its primary function.